src/System.Windows.Forms.Design/src/System/ComponentModel/Design/Serialization/CodeDomLocalizationProvider.LanguageCultureInfoConverter.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Globalization;
+
+namespace System.ComponentModel.Design.Serialization
+{
+    public sealed partial class CodeDomLocalizationProvider
+    {
+        /// <summary>
+        ///  This is a culture info converter that knows how to provide
+        ///  a restricted list of cultures based on the SupportedCultures
+        ///  property of the extender.  If the extender can't be found
+        ///  or the SupportedCultures property returns null, this
+        ///  defaults to the stock implementation.
+        /// </summary>
+        internal sealed class LanguageCultureInfoConverter : CultureInfoConverter
+        {
+            /// <summary>
+            ///  Retrieves the Name for a input CultureInfo.
+            /// </summary>
+            protected override string GetCultureName(CultureInfo culture)
+            {
+                return culture.DisplayName;
+            }
```

Comment:
Let's clean these files as well. 
```suggestion
            protected override string GetCultureName(CultureInfo culture) => culture.DisplayName;
```